conf/broker.conf

```diff
@@ -403,6 +403,9 @@ dispatchThrottlingOnNonBacklogConsumerEnabled=true
 # Max number of entries to read from bookkeeper. By default it is 100 entries.
 dispatcherMaxReadBatchSize=100
 
+# Dispatch messages and execute broker side filters in a per-subscription thread
+dispatcherDispatchMessagesInSubscriptionThread=true
```

Comment:
I am adding this flag only to rollback in case of problem with particular workloads.

We are doing performance testing, I believe that this new behaviour is generally better, not only for the case of server side filters, but especially in these cases:
- multiple subscriptions on the same topic
- concurrent produce and consume